rust/relay/src/proptest.rs

```diff
@@ -0,0 +1,42 @@
+use crate::Binding;
+use proptest::arbitrary::any;
+use proptest::strategy::Just;
+use proptest::strategy::Strategy;
+use proptest::string::string_regex;
+use std::ops::Add;
+use std::time::{Duration, SystemTime};
+use stun_codec::rfc5766::attributes::{ChannelNumber, Lifetime, RequestedTransport};
+use stun_codec::TransactionId;
+
+pub fn transaction_id() -> impl Strategy<Value = TransactionId> {
+    any::<[u8; 12]>().prop_map(|bytes| TransactionId::new(bytes))
+}
+
+pub fn binding() -> impl Strategy<Value = Binding> {
+    transaction_id().prop_map(|id| Binding::new(id))
+}
+
+pub fn udp_requested_transport() -> impl Strategy<Value = RequestedTransport> {
+    Just(RequestedTransport::new(17)) // 17 is udp.
```

Comment:
define a const

Reply:
I did in the production code. This is just for the test but I can define it here too if you want.